authentik/events/models.py

```diff
@@ -170,7 +170,7 @@ class Event(SerializerModel, ExpiringModel):
     context = models.JSONField(default=dict, blank=True)
     client_ip = models.GenericIPAddressField(null=True)
     created = models.DateTimeField(auto_now_add=True)
-    tenant = models.JSONField(default=default_tenant, blank=True)
+    brand = models.JSONField(default=default_brand, blank=True)
```

Comment:
I think we probably want to keep the tenant here instead of switching to brand, maybe adding brand in addition to tenant? Although most HTTP requests already have the domain, and if not we can add it in `from_http()` under `http_request`

Reply:
All events are already tenant-specific, so adding it as a field would not work. If you have a tenant `t_my_tenant` and the `public` tenant (the default one), you'd have two event tables, in two different schemas: `t_my_tenant.authentik_events_event` and `public.authentik_events_event`. I decided to keep the brand in here because it's what was done before.

Reply:
oh yeah d'oh of course, and the tenant info will also be logged correctly through the logging changes below